src/librustc_mir/borrow_check/diagnostics/mod.rs

```diff
@@ -17,6 +19,21 @@ use super::borrow_set::BorrowData;
 use super::MirBorrowckCtxt;
 use crate::dataflow::move_paths::{InitLocation, LookupResult};
 
+mod find_use;
+mod var_name;
+mod region_name;
+mod outlives_suggestion;
+
+crate mod conflict_errors;
+crate mod move_errors;
+crate mod mutability_errors;
+crate mod region_errors;
+crate mod explain_borrow;
```

Comment:
Can you make these private and re-export anything that's not public enough?

Reply:
Woah. Very good call. I thought there were a lot of things, but it turns out there was only one thing that needed to be reexported that wasn't already!